.github/actions/aws-ci/action.yml

```diff
@@ -51,15 +51,19 @@ runs:
       run: poetry run pytest -k "aws" --alluredir=./allure/results/ --clean-alluredir
 
     - name: Upload test results
-      if: ${{ success() || failure() }}
+      if: ${{ always() }}
       uses: actions/upload-artifact@v4
       with:
         name: allure-results-ci-aws
         path: tests/allure/results/
+        overwrite: true
 
-    - if: ${{ (success() || failure()) && inputs.debug == 'true' }}
+    - if: ${{ failure() || cancelled() || inputs.debug == 'true' }}
       name: Upload cloudbeat logs
       uses: actions/upload-artifact@v4
       with:
         name: cloubeat-logs-ci-aws
         path: logs/
+        if-no-files-found: warn
+        retention-days: 5
```

Comment:
@moukoublen I'm not sure if we need to store debug logs for more than a day. Enabling debug mode in the workflow and generating logs is usually a proactive action, and typically, you'll use those logs on the same day.

Reply:
CHanged it to `1`.